daemon/daemon.go

```diff
@@ -427,10 +445,17 @@ func (d *Daemon) Start() {
 		case state.RestartDaemon:
 			d.tomb.Kill(nil)
 		case state.RestartSystem:
-			cmd := exec.Command("shutdown", "+10", "-r", shutdownMsg)
-			if out, err := cmd.CombinedOutput(); err != nil {
-				logger.Noticef("%s", osutil.OutputErr(out, err))
+			// try to schedule a fallback slow reboot already here
+			// in case we get stuck shutting down
+			if err := reboot("+10"); err != nil {
```

Comment:
Should the +10 be based on rebootWaitTimeout global?  Otherwise people will miss this constant when updating the global.